UNRELEASED.md

```diff
@@ -1,6 +1,14 @@
 # Unreleased
 
 > Place your changes below this line.
+**Breaking:**
+ - Backpack/Theme:
+   - `BPKDidChangeNotification` has been renamed `BPKDidChangeThemeNotification`.
```

Comment:
There are more breaking changes than this. 

+ Init renames
+ method renames

Reply:
init is still `initWithThemeDefinition:rootViewController:`
https://github.com/Skyscanner/backpack-ios/pull/316/files#diff-ef5aab263239456e294df6c34c8b99af